src/script/components/Conversation/Conversation.tsx

```diff
@@ -0,0 +1,399 @@
+/* eslint-disable */
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {
+  FC,
+  MouseEvent as ReactMouseEvent,
+  KeyboardEvent as ReactKeyboardEvent,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import {amplify} from 'amplify';
+import {WebAppEvents} from '@wireapp/webapp-events';
+import {groupBy} from 'underscore';
+import cx from 'classnames';
+
+import TitleBar from 'Components/TitleBar';
+import MessagesList from 'Components/MessagesList';
+import InputBar from 'Components/InputBar';
+import Giphy from 'Components/Giphy';
+
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+
+import {Conversation as ConversationEntity} from '../../entity/Conversation';
+
+import {TeamState} from '../../team/TeamState';
+import {UserState} from '../../user/UserState';
+import {PanelViewModel} from '../../view_model/PanelViewModel';
+import {MemberMessage} from '../../entity/message/MemberMessage';
+import {User} from '../../entity/User';
+import {Message} from '../../entity/message/Message';
+import {ContentMessage} from '../../entity/message/ContentMessage';
+import {Text} from '../../entity/message/Text';
+import {safeMailOpen, safeWindowOpen} from 'Util/SanitizationUtil';
+import {ModalsViewModel} from '../../view_model/ModalsViewModel';
+import {t} from 'Util/LocalizerUtil';
+import {UserError} from '../../error/UserError';
+import {DecryptErrorMessage} from '../../entity/message/DecryptErrorMessage';
+import {MotionDuration} from '../../motion/MotionDuration';
+import {getLogger} from 'Util/Logger';
+import {RootContext} from '../../page/RootProvider';
+import {showDetailViewModal} from 'Components/Modals/DetailViewModal';
+
+type ReadMessageBuffer = {conversation: ConversationEntity; message: Message};
+
+interface ConversationListProps {
+  initialMessage?: Message;
```

Comment:
can't this be readonly as well?
```suggestion
  readonly initialMessage?: Message;
```